content/en/docs/contribute/intermediate.md

```diff
@@ -572,21 +573,46 @@ These guidelines are not set in stone and are subject to change.
   in the issue, slash commands used in the comments of the issue, or
   information in the issue text.
 - Some labels are manually added by the person triaging the issue (or the person
-  reporting the issue, if they are a SIG Docs approvers).
-  - `Actionable`: There seems to be enough information for the issue to be fixed
-    or acted upon.
-  - `good first issue`: Someone with limited Kubernetes or SIG Docs experience
-    might be able to tackle this issue.
+  reporting the issue)
+  - The
+    [Help Wanted and Good First Issue Labels](https://github.com/kubernetes/community/blob/master/contributors/guide/help-wanted.md)
+    guide explains `help wanted` and `good first issue` in detail.
+  - The
+    [Issue Triage Guidelines](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#define-priority)
+    define Priority labels in detail.
   - `kind/bug`, `kind/feature`, and `kind/documentation`: If the person who
```

Comment:
These labels aren't ever filled out automatically, so it's up to the person doing triage.

Reply:
@sftim - Really? I didn't know that, so I should clarify that bullet point.

Reply:
If we wanted to make them automatic, we could edit the files inside https://github.com/kubernetes/website/tree/master/.github/ISSUE_TEMPLATE